R/corridor.R

```diff
@@ -103,34 +109,66 @@ river_buffer <- function(river, buffer, bbox = NULL) {
   }
 }
 
-#' Draw a corridor as an estimate of the river valley.
-#'
-#' The valley is drawn on a digital elevation model (DEM) of the area,
-#' see [get_valley()] for details on the implementation.
-#'
-#' @param river A simple feature geometry representing the river
-#' @param dem The DEM of the area as a terra SpatRaster object
-#' @param bbox Bounding box defining the extent of the area of interest
-#'
-#' @return A simple feature geometry
-river_valley <- function(river, dem, bbox = NULL) {
```

Comment:
This function essentially forward a call with the same arguments to `get_valley` - I think we should drop it.